data/tutorials/guides/0tt_05_s_expressions.md

```diff
@@ -0,0 +1,268 @@
+
+The provided examples are intended to be excuted in UTop. As a reminder, you can enter the UTop toplevel with the command `utop`. Here is the initial setup:
+
+```ocaml
+# #require "sexplib";;
+# open Sexplib;;
+```
+
+To manually build S-expressions, we can combine the two constructors `Atom` and `List` presented above. To build the S-expression representing `(1)`: 
+```ocaml
+# Sexp.Atom "1";;
+- : Type.t = Sexplib.Sexp.Atom "1"
+```
+To build the S-expression representing `(1 2)`:
+```ocaml
+# Sexp.List [Atom "1"; Atom "2"];;
+- : Type.t = Sexplib.Sexp.List [Sexplib.Sexp.Atom "1"; Sexplib.Sexp.Atom "2"]
+```
+
+### Printing and Formatting
+
+To display the representation of the S-expressions, formatting functions like `Sexp.pp` are provided:
+
+```ocaml
+# Format.printf "%a\n" Sexp.pp Sexp.(List [List [Atom "1"; Atom "hello"]]);;
+((1 hello))
+- : unit = ()
+```
+
+To convert an S-expression to a string, simply use `Sexp.to_string`:
+```ocaml
+# Sexp.to_string Sexp.(List [List [Atom "1"; Atom "hello"]]);;
+- : string = "((1 hello))"
+```
+
+You can also directly write the representation of an S-expression in a file by using `Sexp.save`:
+```ocaml
+# Sexp.save "myfile" Sexp.(List [List [Atom "1"; Atom "hello"]]);;
+- : unit = ()
+```
+
+## Deserialising
+
+> Deserialisation is the process of extracting a data structure from a series of bytes.
+
+Here it means we can build S-expressions from a storage format, like strings.
+
+### Parsing
+
+To parse an S-expression from a string, one can use `Sexp.of_string`:
+```ocaml
+# Sexp.of_string "((1 hello))";;
+- : Type.t =
+Sexplib.Sexp.List
+ [Sexplib.Sexp.List [Sexplib.Sexp.Atom "1"; Sexplib.Sexp.Atom "hello"]]
+```
+
+Similarly, the function `Sexp.load_sexp` will read an S-expression from a file:
+```ocaml
+# Sexp.load_sexp "myfile";;
+- : Type.t =
+Sexplib.Sexp.List
+ [Sexplib.Sexp.List [Sexplib.Sexp.Atom "1"; Sexplib.Sexp.Atom "hello"]]
+```
+
+### Pattern-Matching
+
+If you don't know the structure of some S-expressions beforehand, it is possible to match on the `Atom` and `List` constructors to lookup for some value. For example:
+
+```ocaml
+# let rec contains_zero = function
+    | Sexp.Atom "0" -> true
+    | Sexp.Atom _ -> false
+    | Sexp.List lx -> List.exists contains_zero lx;;
+val contains_zero : Type.t -> bool = <fun>
+# contains_zero (Sexp.List [Sexp.Atom "1"; Sexp.Atom "hello"]);;
+- : bool = false
+```
+
+## Converting Between OCaml Types and S-Expressions
+
+### Converting OCaml Types Into S-Expressions
+
+To build S-expressions from OCaml predefined types, we can use the `sexp_of_t` functions predefined in this package:
+```ocaml
+# Std.sexp_of_int 0;;
+- : Type.t = Sexp.Atom "0"
```

Comment:
I get:

`- : Type.t = Sexplib0__.Sexp.Atom "0"`

Reply:
@cuihtlauac See if you like the disclaimer I put at the beginning of this section or if we need to tweak it some more.